test/e2e/e2e_gcs_test.go

```diff
@@ -93,17 +102,79 @@ func TestGCSLog(t *testing.T) {
 		if logName == "" {
 			t.Skip("log name not found")
 		}
-		logClient, err := gc.GetLog(context.Background(), &resultsv1alpha2.GetLogRequest{Name: logName})
-		if err != nil {
-			t.Errorf("Error getting Log Client: %v", err)
-		}
-		log, err := logClient.Recv()
-		if err != nil {
-			t.Errorf("Error getting Log: %v", err)
-		}
-		want := "[hello : hello] hello world!"
-		if !strings.Contains(string(log.Data), want) {
-			t.Errorf("Log Data inconsistent got: %s, doesn't have: %s", string(log.Data), want)
+		if err := wait.PollImmediate(1*time.Second, 10*time.Second, func() (done bool, err error) {
```

Comment:
reminder from #712 this is not a long term solution just a validation step

I'll address @sayan-biswas 's feedback first, then make sure I can remove this and still avoid the canceled context on the UpdateLog call